content/swagger/resources/categories/definitions/category.yaml

```diff
@@ -17,7 +17,7 @@ properties:
     x-read-only: true
   position:
     type: integer
-    description: Position of the category in its level, start from 1 (only available on SaaS platforms and when query parameter "with_position" is set to "true")
+    description: Position of the category in its level, start from 1 (only available since 7.0 and when query parameter "with_position" is set to "true")
```

Comment:
```suggestion
    description: Position of the category in its level, start from 1 (only available since the 7.0 version and when query parameter "with_position" is set to "true")
```